docs/ibis-for-dplyr-users.ipynb

```diff
@@ -0,0 +1,774 @@
+    "\n",
+    "With dplyr:\n",
+    "\n",
+    "```r\n",
+    "starwars |>\n",
+    "  filter(skin_color == \"light\")\n",
+    "```\n",
+    "\n",
+    "In Ibis:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starwars.filter(_.skin_color == \"light\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In dplyr, you can specify multiple conditions separated with `,` that are then combined with the `&` operator:\n",
+    "\n",
+    "```r\n",
+    "starwars |>\n",
+    "  filter(skin_color == \"light\", eye_color == \"brown\")\n",
+    "```\n",
+    "\n",
+    "In Ibis, you can do the same by putting multiple conditions in a list:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starwars.filter([_.skin_color == \"light\", _.eye_color == \"brown\"])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In previous code, we used the `_` helper we imported earlier. The `_` is shorthand for the table we referenced at the beginning of the expression (in this case, `starwars`). We could have also written the more verbose form,\n",
+    "\n",
+    "```python\n",
+    "starwars.filter([starwars.skin_color == \"light\", starwars.eye_color == \"brown\"])\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to combine multiple conditions, in dplyr, you could do:\n",
+    "\n",
+    "```r\n",
+    "starwars |>\n",
+    "  filter(\n",
+    "      (skin_color == \"light\" & eye_color == \"brown\") |\n",
+    "       species == \"Droid\"\n",
+    "  )\n",
+    "```\n",
+    "\n",
+    "In Ibis, this would be:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starwars.filter(\n",
+    "    ((_.skin_color == \"light\") & (_.eye_color == \"brown\")) |\n",
+    "    (_.species == \"Droid\")\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Sorting your data with order_by()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To sort a column, dplyr has the verb `arrange`. For instance, to sort the column `height` using dplyr:\n",
+    "\n",
+    "```r\n",
+    "starwars |>\n",
+    "   arrange(height)\n",
+    "```\n",
+    "\n",
+    "In Ibis:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starwars.order_by(_.height)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You might notice that while dplyr puts missing values at the end, Ibis places them at the top.\n",
```

Comment:
> `Ibis places them at the top`

Does this vary depending on what backend you use?

Reply:
@gforsyth do you happen to know? I'm guessing yes based on what @cpcloud said in review,

> Most databases allow you to choose null ordering either as a global configuration option, in the expression itself (e.g., NULLS LAST) or both. So far nobody has asked for this functionality so we haven't implemented it.

Reply:
Yeah, I'm pretty sure this is backend-dependent.  We haven't yet exposed a way to configure it (and I don't know that there's a consensus on whether `FIRST` or `LAST` is a "better" default)

Reply:
Thanks @gforsyth, I clarify the text here.